functest/deploy_test.go

```diff
@@ -276,12 +276,12 @@ func TestNoCheckFlag(t *testing.T) {
 	imageTag := sandbox.ProjectName + "/noflag"
 
 	t.Log("Running appsody deploy...")
-	output, err := cmdtest.RunAppsody(sandbox, "deploy", "-t", imageTag, "--dryrun", "--no-check")
+	output, err := cmdtest.RunAppsody(sandbox, "deploy", "-t", imageTag, "--dryrun", "--no-operator-check")
 	if err != nil {
 		t.Log("WARNING: deploy dryrun failed.")
 	}
 
-	if !strings.Contains(output, "kubectl get pods -o=jsonpath='{.items[?(@.metadata.labels.name==\"appsody-operator\")].metadata.namespace}' -n") {
+	if !strings.Contains(output, "kubectl get pods \"-o=jsonpath='{.items[?(@.metadata.labels.name==\"appsody-operator\")].metadata.namespace}'\" -n") {
```

Comment:
Do we want to make the operator test changes in this PR?

Reply:
I corrected the flag name `no-check` --> `no-operator-check` in the deploy test whilst making these changes. True, it could go in another PR but I thought since its a one line change I'd include it here. As for the other tests I've changed, I've updated them to point to the v2 index, `dev.local-index.yaml` instead of the v1 index, `index.yaml`.